lib/src/dexode/EventBus.cpp

```diff
@@ -30,7 +32,7 @@ std::size_t EventBus::processLimit(const std::size_t limit)
 
 	{
 		std::lock_guard writeGuard{_mutexStreams};
-		if(not _eventStreams.empty())
+		if(!_eventStreams.empty())
```

Comment:
no change, I use `not` because it helps to read code. I had a lot of errors in past because I didn't spot `!` mark.